### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Fluffy-Bean** [Fluffy-Bean/TastyBites#2]
FireFox reports a few accessibility issues, mainly contrast and missing label names.

## Contrast and Colorblindness
<details><summary>Example images</summary>

![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/71cf4817-d9d4-456d-85b6-519447150a91)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/72ac565e-2d51-4bbe-bce3-904f0d5cf5fe)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/d6204828-0822-486b-8c07-f5631c42a8bb)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/8a4018ea-50ac-4d39-a27a-283c113907d7)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/da39ed3d-8bfa-4099-8c68-cc13a2c07eeb)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/34cefb44-b625-428c-9378-d476a2646920)

</details>

Some of these issues are related to colorblindness or loss of contrast in vision, which can be fixed with an optional accessibility panel. Others are triggered by normal use, specifically the green on the navigation bar, or the red error color on the background. Either I adjust the colors, or provide options to increase contrast levels based on a setting.

## Tabbing order
While Firefox doesn't report on this (I haven't tested Chrome lighthouse yet) this looks like an issue to me.

<details><summary>Example images</summary>

![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/06f0a34b-8ef1-447b-a854-4140431ea690)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/de67bbef-686d-4deb-aaeb-5f8d9de7b9ab)
![image](https://github.com/Fluffy-Bean/TastyBites/assets/86935712/db60955b-1d60-420f-b4a2-da3452363208)

</details>

This could be fixed by allowing the user to tab from section to section, or "island to island" of content, then allow them to navigate to the correct element through the use of arrow keys. Though I would need to read on the best practices for this, and how to them implement this within Svelte.
